src/animation.js

```diff
@@ -126,7 +126,7 @@ export class Animation {
  * animation.
  * @implements {IThenable}
  */
-class AnimationPlayer {
+export class AnimationPlayer {
```

Comment:
closure compiler won't see the type if we don't explicitly don't export it unfortunately.